src/examples/pongxr/scene.js

```diff
@@ -0,0 +1,200 @@
+/*****
```

Comment:
This is an awesome example to work from!

Reply:
sweet, thanks! Trying to tie physics, networking and XR controllers all together while keeping things readable... it's an interesting challenge:)